website/blog/2025-12-04-release-1.24.md

```diff
@@ -0,0 +1,257 @@
+
+<ThemedImage
+alt="Containers Multiple Connections"
+sources={{
+    light: require('./img/podman-desktop-release-1.24/containers-multiple-connections-light.png').default,
+    dark: require('./img/podman-desktop-release-1.24/containers-multiple-connections-dark.png').default,
+  }}
+/>
+
+<ThemedImage
+alt="Multiple Podman Machine"
+sources={{
+    light: require('./img/podman-desktop-release-1.24/multiple-podman-machine-light.png').default,
+    dark: require('./img/podman-desktop-release-1.24/multiple-podman-machine-dark.png').default,
+  }}
+/>
+
+### Select Build Targets in Containerfile
+
+Building images with multiple targets in your Containerfile is now simpler. On the Build Image page, Podman Desktop now detects automatically the available stages, and a new dropdown menu lets you select the desired build target. This feature is equivalent to the [`--target` option](https://docs.podman.io/en/v3.0/markdown/podman-build.1.html#target-stagename) in the Podman CLI and simplifies complex multi-stage builds.
+
+<ThemedImage
+alt="Dropdown Multi-Target Containerfile"
+sources={{
+    light: require('./img/podman-desktop-release-1.24/dropdown-multi-target-containerfile-light.png').default,
+    dark: require('./img/podman-desktop-release-1.24/dropdown-multi-target-containerfile-dark.png').default,
+  }}
+/>
+
+### Podman Kube Play YAML Deployment Enhanced with 'replace' Option
+
+For users applying YAML files via Podman Kube Play ([https://docs.podman.io/en/v5.5.1/markdown/podman-kube-play.1.html](https://docs.podman.io/en/v5.5.1/markdown/podman-kube-play.1.html)), support for the `replace` option has been added. This offers more flexibility when updating resources, allowing for smoother deployments and modifications.
+
+<ThemedImage
+alt="Podman Kube Play Replace Option"
+sources={{
+    light: require('./img/podman-desktop-release-1.24/kube-play-replace-option-light.png').default,
+    dark: require('./img/podman-desktop-release-1.24/kube-play-replace-option-dark.png').default,
+  }}
+/>
+
+### New Task Manager is Now Default
+
+The updated Task Manager is now the default experience in Podman Desktop, replacing the older Legacy Task Manager. This change provides a more efficient way to monitor and manage your ongoing tasks.
+
+<ThemedImage
+alt="New Task Manager"
+sources={{
+    light: require('./img/podman-desktop-release-1.24/new-task-manager-light.png').default,
+    dark: require('./img/podman-desktop-release-1.24/new-task-manager-dark.png').default,
+  }}
+/>
+
+### Updated GPU Setup Instructions
+
+We have updated the `dnf` commands and clarified the GPU setup instructions for users utilizing GPU capabilities. These changes aim to make the configuration process for GPU-accelerated environments more accurate.
+
+### Performance Improvement for Podman Version Check
```

Comment:
I think...this is only for windows, we should specify that. Also I would move this higher, there  was a significant amount of bugs/issues related to this